api/dbadapters/bigquery.ts

```diff
@@ -467,20 +467,24 @@ DELETE \`${CACHED_STATE_TABLE_NAME}\` WHERE target IN (${allActions
                     reject(new Error(jobMetadata.status.errorResult.message));
                     return;
                   }
-                  const queryData = {
+                  const queryData: IExecutionResult = {
                     rows: results,
                     metadata: {
                       bigquery: {
-                        jobId: jobMetadata.jobReference.jobId,
-                        totalBytesBilled: Long.fromString(
-                          jobMetadata.statistics.query.totalBytesBilled
-                        ),
-                        totalBytesProcessed: Long.fromString(
-                          jobMetadata.statistics.query.totalBytesProcessed
-                        )
+                        jobId: jobMetadata.jobReference.jobId
```

Comment:
Bit tidier shorten with ternary IMO, don't mind
```suggestion
                        totalBytesBilled: !!jobMetadata.statistics.query.totalBytesBilled ? jobMetadata.statistics.query.totalBytesBilled : undefined
```

Reply:
it's semantically different, though. the ternary form results in the `totalBytesBilled` key now existing on the object, even if the condition evaluates falsey.